utils.js

```diff
@@ -159,7 +183,7 @@ module.exports = {
 
       return tockSlackUsers;
     };
-    const postEphemeralMessage = (message) => {
+    const postEphemeralMessage = async (message) => {
       webAPI.chat.postEphemeral(message);
```

Comment:
Will callers of `postEphemeralMessage()` ever want to know when the promise from `webAPI.chat.postEphemeral()` resolves? Is so, should this be `await`ing it so this doesn't resolve immediately ?

Reply:
They might in some future world, but given Charlie's trajectory, I wouldn't expect its needs to change much. That said, it seems like it's a good practice to actually make this thing asynchronous, since that's what it claims to be.

(It previously was not, and I changed it when I was running into a timing issue in the tests... but that issue turned out to be something else and I just forgot to remove the `async` keyword here. Still, it is an asynchronous web call, so there's no harm in wrapping it up as such.)

Reply:
Done! 😄